src/js/tabs/configuration.js

```diff
@@ -905,12 +905,9 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             $('div.disarm').hide();
         }
 
-        $('._smallAngle').hide();
+        $('._smallAngle').toggle(semver.gte(CONFIG.apiVersion, "1.37.0"));
         if(semver.gte(CONFIG.apiVersion, "1.37.0")) {
             $('input[id="configurationSmallAngle"]').val(ARMING_CONFIG.small_angle);
-            if (SENSOR_CONFIG.acc_hardware !== 1) {
```

Comment:
Just for my understanding: We do not need to check for `SENSOR_CONFIG.acc_hardware !== 1` any more here because we are already hiding the entire block if no ACC is present?

Reply:
Yes, right, we already have a function for hide entire block based in accswitch checking.